antenna-documentation/src/site/markdown/processors/manifest-resolver.md

```diff
@@ -1,16 +1,17 @@
 ## Manifest resolver
-The Manifest resolver checks the first element of the list of path names for each artifact. If a JAR file is found 
-and it contains a MANIFEST.MF file with the properties `Bundle-SymbolicName` and `Bundle-Version`, it will add a 
-`BundleCoordinates` fact to the artifact. It's deactivated in the default antenna workflow configuration, i.e. it 
-has to be activated in the workflow.xml. 
- 
+The Manifest resolver checks for each artifact if a JAR file is found as fact
+and the jar filecontains a MANIFEST.MF file with the properties `Bundle-SymbolicName` and `Bundle-Version`.
```

Comment:
space in filecontains.

Reply:
p2 is still supported, but not as good as in the past, so having checks that still find p2 components makes sense.